scripts/generate-cov-report.sh

```diff
@@ -54,13 +59,42 @@ show_report() {
 }
 
 generate_html_report() {
-    mkdir -p ./target/coverage/html/
-    grcov . --binary-path ./target/debug/deps/ -s . -t html --branch --ignore-not-existing --ignore '../*' --ignore "/*" -o target/coverage/html
+    mkdir -p ./target/coverage/
+    grcov \
+          **/${LLVM_PROFILE_PATH} \
+          **/**/${LLVM_PROFILE_PATH} \
+          --binary-path ./target/debug \
+          --source-dir . \
+          --output-type html \
+          --branch \
+          --ignore-not-existing \
+          --ignore "**/build.rs" \
+          --ignore "**/tests/*" \
+          --ignore "**/examples/*" \
+          --ignore "**/benchmarks/*" \
+          --ignore "**/target/*" \
+          --ignore "**/.cargo/*" \
+          --output-path ./target/coverage/html
```

Comment:
I kept the `targe/coverage` path but `cargo clean` does not know about this path and therefore does not clean this folder up. Shall we move everything to `target/debug`?

Reply:
Yes please.